customer-commons/src/test/java/no/unit/nva/customer/model/SecretManagerDoiAgentDaoTest.java

```diff
@@ -42,10 +38,13 @@ void fromJsonFailing() {
 
     @Test
     void jacocoTestForTestCoverageThisIsAlreadyTestedElsewhere() {
-        var customerDto = createSampleCustomerDao().toCustomerDto();
-        var doiAgentDto = customerDto.getDoiAgent();
-        var secretManagerDoiAgentDao = new SecretManagerDoiAgentDao(randomUri(),doiAgentDto);
+        var doiAgentDto = createSampleCustomerDao().toCustomerDto()
```

Comment:
change name to shouldUpdate… as above. The fact that the unit under test is tested elsewhere can be mentioned in a comment